examples/deployment/kubernetes/trillian-log-signer-deployment.yaml

```diff
@@ -24,14 +24,16 @@ spec:
       - command: ["/go/bin/trillian_log_signer",
         "$(STORAGE_FLAG)",
         "--storage_system=$(STORAGE_SYSTEM)",
-        "--quota_system=noop",
+        "--etcd_servers=trillian-etcd-cluster-client:2379",
+        "--quota_system=etcd",
```

Comment:
You sure you want to turn on quota as well as mastership election?  (If so, should there be a corresponding change to the log-server-deployment?)